### DISCUSSION TITLE
[BUG] No response when prompting

### DISCUSSION THREAD

**Discussion by Gako358** [chep/copilot-chat.el#4] [Support]
# Description
I have managed to setup `copilot-chat`, but there is no response.
I am using the `copilot` from `https://github.com/copilot-emacs/copilot.el`

My config:

```lisp
  (let
      (
       (copilot-chat-dir "~/.emacs.d/emacsCopilotChat")
       (copilot-chat-file "~/.emacs.d/emacsCopilotChat/copilot-chat.el"))
    ;; Check if the copilot-chat.el file exists
    (when
        (file-exists-p copilot-chat-file)
      ;; Add the directory to the load-path
      (add-to-list 'load-path copilot-chat-dir)
      ;; Try to load the copilot-chat module and catch any errors
      (condition-case err
          (progn
            (use-package copilot-chat))
        (error
         (message "Failed to load copilot-chat: %s" err)))))

```

All the commands are available, but no output from chat, i do not know how to debug it anymore either, so if anyone know?

**Comment by chep** [comment]
Do you have anything in *messages* buffer ?

**Comment by Gako358** [comment]
only thing relevant is :

```
url-retrieve-internal: Wrong number of arguments: #<subr url-https>, 1
```

When I submit a text, from the `copilot-chat-prompt`, it then shows in the `copilot-chat`. but no response.

**Comment by Gako358** [comment]
and when i journal emacs.service i get one warning, that i relevant:

```
.emacs.d/emacsCopilotChat/copilot-chat.el: Warning: (lambda nil \...) quoted with ' rather than with #'
```

**Comment by chep** [comment]
Did the authentication work?
Can you update your files, kill emacs, remove `~/.cache/copilot-chat/token` and `~/.config/copilot-chat/github-token`, start emacs again, call `copilot-chat-display`, enter something and send it (`C-c RET`) ? 
You should be asked to authenticate in a browser with a unique code.

**Comment by Gako358** [comment]
I do not have any `copilot` in `~/.cache` and in `~/.config` i only have `github-copilot`

**Comment by chep** [comment]
Variables `copilot-chat-token-cache` and `copilot-chat-github-token-file` hold tokens filename. If the files do not exist, then something went wrong…
I need to improve error management.

**Comment by Gako358** [comment]
Thanks for the reply.
